test/cli/test_cli_functions.sh

```diff
@@ -634,9 +634,61 @@ function account_cycle {
     account_reset_password "admin@noobaa.io"
     # testing nsfs accounts
     # account_nsfs_cycle TODO re-enable.
+    # update account default resource
+    account_update ""
+    # update crd account default resource
+    account_update "crd"
     echo_time "✅  noobaa account cycle is done"
 }
 
+function account_update {
+    local crd=${1}
+    echo_time "💬  Checking for ${crd} account updation"
+    local res="backing-store-upd"
```

Comment:
This variable is meant for new_default_resource and not res. 
we can change the name to `new_default_resource` and change all the calls for it accordingly
```suggestion
    local new_default_resource="backing-store-upd"
```